.github/workflows/julia_ci.yml

```
@@ -13,7 +13,7 @@ on:
         type: string
       test_versions:
         description: "What versions of julia to test, a JSON array of strings"
-        default: "['1']"
+        default: "['1.10.5', '1.11.0']" # 1.10.5 is LTS
```

```suggestion
        default: "['1.10.5', '1']" # 1.10.5 is LTS
```

This should do LTS and latest

---

this is, _if_, we want to try and support LTS again. We tried before and basically no one in the Julia community supports LTS in an intelligent way and nearly no one uses it after a while. Previously I wanted us to test against LTS but it became pretty hard as more breaking changes happened